src/main/kotlin/assertk/assertions/inputstream.kt

```diff
@@ -0,0 +1,62 @@
+package assertk.assertions
+
+import assertk.Assert
+import assertk.assertions.support.expected
+import java.io.InputStream
+
+fun Assert<InputStream>.hasSameContentAs(other: InputStream) {
```

Comment:
I don't know if it was there when you first started looking, but I added docs to all of the assertions. Can you do the same? ex: https://github.com/willowtreeapps/assertk/blob/master/src/main/kotlin/assertk/assertions/boolean.kt

Reply:
Yep, when I forked it, I couldn't find them. I will add them.